File: frontend/src/modelos/Modelo.ts
export default class Modelo{
    public id!: number
    public nome: string;
    public descricao: string;
    public modelo: string;
    constructor(nome: string, descricao: string, modelo: string){
        this.nome = nome;
        this.descricao = descricao;
        this.modelo = modelo;
    }
}

File: demo/src/main/java/com/crud/modelo/entidade/Modelo.java
package com.crud.modelo.entidade;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;

@Data
@Entity
public class Modelo {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	@Column
	private String nome;
	@Column
	private String descricao;
	@Column
	private String modelo;
}
